### ISSUE TITLE
Alt text should be wrapped

### ISSUE THREAD

**Issue by zmc** [elk-zone/elk#2509]
See this post as an example: https://elk.zone/hachyderm.io/@nocontexttrek@mastodon.social/111562653230279892

The full alt text is: "DS9 scene. Dax is pictured on station in quarters. It's a mechanical feeling room, with a replicator machine and station lighting in the background. She's standing in a nightgown, and looks hungover and tired. She's holding a cup of coffee, and her other hand is held up as if she's about to say something/has an idea. In the second picture, she has decided against whatever that was and instead is having another sip of her coffee."

As you can see in the screenshot, only this is shown:
"DS9 scene. Dax is pictured on station in quarters. It's a mechanical feeling room, with a replicator machine and station lighting in the background. She's standing in a nightgown, and looks h..."
<img width="1395" alt="shortalttext" src="https://github.com/elk-zone/elk/assets/58307/1be94ab9-8ce6-4baa-a1e6-9398c0822ba4">

**Comment by HawtinZeng** [comment]
You can hover your mouse on the description, you will see the full text, the abbreviation is for better text layout.Shown in the screenshot below:
![image](https://github.com/elk-zone/elk/assets/73981739/2a94572e-eb5f-42a7-a773-c5edfbc77f1e)

**Comment by zmc** [comment]
Oh that's good enough for me at least, thanks

**Comment by mojoaxel** [comment]
This mouse-over info text does not work on mobile. Here the problem still exists!

![Screenshot_20241119-221458.png](https://github.com/user-attachments/assets/32fd6055-ab8e-47f8-870f-ba5285584811)
